Week1/Homework/insert-values.js

```diff
@@ -0,0 +1,42 @@
+var mysql = require('mysql');
+var connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world',
+});
+
+connection.connect();
+var insert_queries = [
+  "insert into continent(name) values ('Europe')",
+  "insert into continent (name) values ('Asia')",
+  "insert into continent (name) values ('South America')",
+  "insert into continent (name) values ('North America')",
+  "insert into continent (name) values ('Africa')",
+  "insert into continent (name) values ('Autralia')",
+  "insert into continent (name) values ('Kenorland')",
+  "insert into Country(name, population, service_area, continentID) values ('Netherlands', 17000000, 41543,1)",
+  "insert into Country(name, population, service_area, continentID) values ('France', 67348000, 547030,1)",
+  "insert into Country(name, population, service_area, continentID) values ('Germany', 82800000, 357168,1)",
+  "insert into Country(name, population, service_area, continentID) values ('Norway', 5295619, 385203,1)",
+  "insert into Country(name, population, service_area, continentID) values ('Sweden', 10151588, 450295,1)",
+  "insert into Country(name, population, service_area, continentID) values ('Switzerland', 8401120, 41285,1)",
+  "insert into Country(name, population, service_area, continentID) values ('Turkey', 80810525, 783356	,1)",
+  "insert into City (name, population, countryID) values('Utrecht','345000',1)",
```

Comment:
This works. You can also pull out the data part and use this helper https://github.com/mysqljs/mysql#escaping-query-identifiers